src/Reports/SiteSummary.php

```diff
@@ -23,7 +23,18 @@ public function description()
         );
     }
 
-    public function sourceRecords($params)
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+
+        $fields->insertAfter('ReportDescription', new LiteralField(
+            'Version', '<p><strong>' . _t(__CLASS__ . '.VERSION', 'Version: ') . $this->resolveCmsVersion() . '</strong></p>'
+        ));
```

Comment:
The only concern I have with this is that you're adding the field _after_ the extension point, so the update method won't affect it for other devs who might want to remove or modify this field.

Can you switch to using the `beforeUpdateCMSFields` callback?

```php
$this->beforeUpdateCMSFields(function (FieldList $fields) {
    $fields->insertAfter...
});
return parent::getCMSFields();
```

Reply:
Perfect, cheers